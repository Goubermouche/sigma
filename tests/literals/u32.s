i32 main() {
	u32 min = 0;
	u32 max = 4294967295;

	printf("%u %u\n", min, max);

	u32 overflow = 4294967296;
	u32 underflow = -1;

	printf("%u %u\n", overflow, underflow);

	ret 0;
}
