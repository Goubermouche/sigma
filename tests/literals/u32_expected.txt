0 4294967295
0 4294967295
